pkg/entity/flag.go

```diff
@@ -36,23 +36,23 @@ type FlagEvaluation struct {
 // Preloads just the tags
 func PreloadFlagTags(db *gorm.DB) *gorm.DB {
 	return db.Preload("Tags", func(db *gorm.DB) *gorm.DB {
-		return db.Order("id ASC")
+		return db.Order("id")
 	})
 }
 
 // PreloadSegmentsVariantsTags preloads segments, variants and tags for flag
 func PreloadSegmentsVariantsTags(db *gorm.DB) *gorm.DB {
-	return db.
+	return db.Debug().
 		Preload("Segments", func(db *gorm.DB) *gorm.DB {
 			return PreloadConstraintsDistribution(db).
-				Order("rank ASC").
-				Order("id ASC")
+				Order("rank").
```

Comment:
`"rank ASC"` was not escaped correctly in mysql8 due to the new keyword `RANK` function. Here we can just remove `ASC` and let gorm handle the field name escape correctly.